src/components/Checkbox/Checkbox.js

```diff
@@ -126,6 +126,8 @@ Checkbox.defaultProps = {
   indeterminate: false,
 };
 
-export default React.forwardRef((props, ref) => (
-  <Checkbox {...props} forwardRef={ref} />
-));
+const forwardRef = (props, ref) => <Checkbox {...props} forwardRef={ref} />;
+
+forwardRef.displayName = 'Checkbox';
+
+export default forwardRef;
```

Comment:
Might need to be:

```suggestion
export default React.forwardRef(forwardRef);
```

?

Reply:
Oops you are right - Made changes as such.